ledger/src/blockstore_meta.rs

```diff
@@ -125,10 +144,6 @@ impl ShredIndex {
             self.set_present(idx, present);
         }
     }
-
-    pub fn largest(&self) -> Option<u64> {
```

Comment:
This function wasn't used anywhere so I deleted it. If we did need to keep this function around for some reason, BitVector implementation wouldn't be super pretty. We'd have to do something like this in in `set_present()`:
```
            ...
            self.index.set(index, presence);
            if presence {
                self.num_present += 1;
                // Compute new largest
                largest = cmp::max(largest, index);
            } else {
                self.num_present -= 1;
                // Scan for new largest, starting at index and running backwards
            }
```